src/babashka/nrepl/impl/server.clj

```diff
@@ -46,7 +51,14 @@
                 (set! *1 value)
                 (utils/send o (utils/response-for msg
                                                   {"ns" (vars/current-ns-name)
-                                                   "value" (pr-str value)}) opts)
+                                                   "value" (if nrepl-pprint
+                                                             (let [pprint-fn (get nrepl-pprint pretty-print-fns-map)]
+                                                               (if (not (nil? pprint-fn))
+                                                                 (pprint-fn value)
+                                                                 (do
+                                                                   (println "Pretty-Printing is only supported for clojure.core/prn and clojure.pprint/pprint.")
```

Comment:
I would only print this when debugging. See other spots where we are printing debug info.